inst/validation/pmforest-stories.yaml

```diff
@@ -77,7 +77,17 @@ PLOT-S011:
   - PMF-PLOT-014
   - PMF-PLOT-015
   - PMF-PLOT-016
-DATA-S001:
+  - PMF-PLOT-023
+PLOT-S012:
+  name: Formatting options, ggplot themes
+  description: As a user, I would like to have be able to change the shape and size
+    of the median value. I would also like to modify the default ggplot theme.
+  ProductRisk: medium
+  tests:
+  - PMF-PLOT-020
+  - PMF-PLOT-021
+  - PMF-PLOT-022
+DATA-S002:
```

Comment:
This is fine for this time, but going forward let's not re-number stories. It will make it harder to map one set of validation docs to the next. And the ordering doesn't really matter anyway.